control-plane/agents/src/bin/ha/node/main.rs

```diff
@@ -40,8 +43,14 @@ struct Cli {
     node_name: String,
 
     /// IP address and port for the ha node-agent to listen on.
-    #[clap(short, long, default_value = DEFAULT_NODE_AGENT_SERVER_ADDR)]
-    grpc_endpoint: SocketAddr,
+    #[clap(short, long = "grpc-endpoint")]
```

Comment:
Does the short become `d` or `g` in this case?

Reply:
Ah good spot, yeah that should be made explicit too

Reply:
fixed this